src/package.lisp

```diff
@@ -175,4 +175,7 @@
    #:defcvar
    #:get-var-pointer
    #:foreign-symbol-pointer
+   ;; *errno* and strerror are only available when the :cffi/errno ASDF system is loaded
```

Comment:
when loading errno.lisp on a system that is not supported, it signals a continuable error at compile time and a hard error at runtime. we could include it in the main CFFI system and change the compile time signal to a warning.